Snakefiles/workflow/scripts/predict.py

```diff
@@ -19,39 +19,36 @@ class formatter(argparse.ArgumentDefaultsHelpFormatter, argparse.RawTextHelpForm
     parser.add_argument('--enhancers', required=True, help="Candidate enhancer regions. Formatted as the EnhancerList.txt file produced by run.neighborhoods.py")
     parser.add_argument('--genes', required=True, help="Genes to make predictions for. Formatted as the GeneList.txt file produced by run.neighborhoods.py")
     parser.add_argument('--outdir', required=True, help="output directory")
-    parser.add_argument('--window', type=int, default=5000000, help="Consider all candidate elements within this distance of the gene's TSS when computing ABC scores. This was a parameter optimized during development of ABC and should not typically be changed.")
+    parser.add_argument('--window', type=int, default=5000000, help="Make predictions for all candidate elements within this distance of the gene's TSS")
     parser.add_argument('--score_column', default='ABC.Score', help="Column name of score to use for thresholding")
-    parser.add_argument('--threshold', type=float, required=True, default=.022, help="Threshold on ABC Score (--score_column) to call a predicted positive. Note that the threshold will need to be adjusted based on the combination of input datasets used.")
+    parser.add_argument('--threshold', type=float, required=True, default=.022, help="Threshold on ABC Score (--score_column) to call a predicted positive")
     parser.add_argument('--cellType', help="Name of cell type")
     parser.add_argument('--chrom_sizes', help="Chromosome sizes file")
 
     #hic
     #To do: validate params
     parser.add_argument('--HiCdir', default=None, help="HiC directory")
     parser.add_argument('--hic_resolution', type=int, help="HiC resolution")
+    parser.add_argument('--tss_hic_contribution', type=float, default=100, help="Weighting of diagonal bin of hic matrix as a percentage of the maximum of its neighboring bins")
     parser.add_argument('--hic_pseudocount_distance', type=int, default=1e6, help="A pseudocount is added equal to the powerlaw fit at this distance")
     parser.add_argument('--hic_type', default = 'juicebox', choices=['juicebox','bedpe', 'avg'], help="format of hic files")
-    parser.add_argument('--hic_is_doubly_stochastic', action='store_true', help="If hic matrix is already doubly stochastic, can skip this step")
+    parser.add_argument('--hic_is_doubly_stochastic', action='store_true', help="If hic matrix is already DS, can skip this step")
 
     #Power law
-    parser.add_argument('--scale_hic_using_powerlaw', action="store_true", help="Quantile normalize Hi-C values using powerlaw relationship. This parameter will rescale Hi-C contacts from the input Hi-C data (specified by --hic_gamma and --hic_scale) to match the power-law relationship of a reference cell type (specified by --hic_gamma_reference)")
```

Comment:
Copy over these various changes to the help text, which I believe I made on the encode branch to better explain the parameters